agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/AzureMonitorLogProcessor.java

```diff
@@ -20,6 +20,10 @@ public void onEmit(Context context, ReadWriteLogRecord logRecord) {
       return;
     }
     ReadableSpan readableSpan = (ReadableSpan) currentSpan;
+    currentSpan.setAttribute(
+        "applicationinsights.internal.exception_already_logged",
```

Comment:
Let's apply this to the LocalRootSpan instead of the current span, in case we're down a level

also, I think this name may make more sense

```suggestion
        "applicationinsights.internal.exception_logged",
```

Reply:
Perhaps the attribute key could be defined in only one place (not both in the Azure exporter and the Java agent). Perhaps in the Azure exporter and in a follow up PR?

Reply:
@heyams I think you missed my comment above:

> Let's apply this to the LocalRootSpan instead of the current span, in case we're down a level

this will affect the smoke test with controller spans enabled

Reply:
> Perhaps the attribute key could be defined in only one place (not both in the Azure exporter and the Java agent). Perhaps in the Azure exporter and in a follow up PR?

it's defined in single place. there is a todo right next to it.

Reply:
> @heyams I think you missed my comment above:
> 
> > Let's apply this to the LocalRootSpan instead of the current span, in case we're down a level
> 
> this will affect the smoke test with controller spans enabled

Updated by setting the new attribute in the LocalRootSpan.